playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/post.yml

```diff
@@ -37,7 +37,7 @@
   - name: Update router image to current version
     when: all_routers.rc == 0
     command: >
-      {{ oc_cmd }} patch dc/{{ item['labels']['deploymentconfig'] }} -p
+      {{ oc_cmd }} patch dc/{{ item['labels']['deploymentconfig'] }} -n {{ item['namespace'] }} -p
```

Comment:
I'm not sure this is the intended behavior, the above get uses `--all-namespaces` and not `-n default`.

This probably needs to be updated to somehow get the namespace from `haproxy_routers` instead.